nixos/modules/programs/zsh/zsh.nix

```diff
@@ -143,6 +174,26 @@ in
 
         ${cfge.interactiveShellInit}
 
+        ${optionalString (cfg.enableOhMyZsh)
```

Comment:
In an external module this would be then moved to: programs.zsh.interactiveShellInit